pom.xml

```
@@ -153,8 +153,8 @@
     <modules>
         <module>build/enforcer</module>
         <module>bom/public</module>
-        <module>bom/build</module>
         <module>build/config</module>
+        <module>build/parents/build</module>
```

Technically build/config needs build/parents/build at the moment...

```suggestion
        <module>build/parents/build</module>
        <module>build/config</module>
```